### PR TITLE
First draft of release notes for week of 9-21-15

### PR DESCRIPTION
**Pull request by srpearce**
@explorerleslie @shnayder @cptvitamin Please review for content. There are a couple of items that I'm not sure we should include; I was kind of scrounging. Also, @shnayder, let me know if you can think of a way to make the Analytics announcement slightly less anemic - do we have an estimate of how much faster signing in is, percentage-wise?

@mhoeber Just letting you know I've completed a draft - I think we should let the product owners review for content before doing an editorial pass.

### CODE REVIEW

File: en_us/release_notes/source/2015/studio/studio_0921_2015.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+* For edx.org courses, course teams can no longer update the **Enrollment End
+  Date** and **Enrollment End Time** fields on the **Schedule & Details**
+  page. Course teams can contact their edX program managers to update these
+  fields. (TNL-2694)
+
+* On the Studio sign-in page, users can now use the keyboard to select the
+  Studio **Forgot password** link. Previously, users had to use a mouse to
+  select this link.
```
Comment (cptvitamin) [inline en_us/release_notes/source/2015/studio/studio_0921_2015.rst:9]: :+1:   all good on the a11y front

File: en_us/release_notes/source/2015/studio/studio_0921_2015.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+
+* For edx.org courses, course teams can no longer update the **Enrollment End
+  Date** and **Enrollment End Time** fields on the **Schedule & Details**
+  page. Course teams can contact their edX program managers to update these
```
Comment (mhoeber) [inline en_us/release_notes/source/2015/studio/studio_0921_2015.rst]: program manager -> edX Partner Manager.  We had to make this change globally last week; I neglected to search release notes

File: en_us/release_notes/source/2015/lms/lms_0921_2015.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+For CCX courses, instructors now receive an error message if they use an
+invalid or unregistered email address to add a user to a course.
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/lms/lms_0921_2015.rst]: Do we have docs on CCX or is it still all private? If it's all undocumented, we should remove this. I just don't want confusion on what the heck CCX is.

**Comment by explorerleslie** [comment]
:+1:

File: en_us/release_notes/source/2015/lms/lms_0921_2015.rst (hunk 3 above)
Comment (srpearce) [inline en_us/release_notes/source/2015/lms/lms_0921_2015.rst]: We have CCX documentation for Open edX - I'll add a bit more info and a link to the docs here.

**Pull request merged**